index.js

```diff
@@ -48,17 +72,18 @@ CompassCompilerPlugin.prototype.toTree = function(tree, inputPath, outputPath, i
     var outputFileName = path.basename(outputPaths[file], extension) + extension; // new name for asset
     var outputDir = removeLeadingSlash(path.dirname(outputPaths[file])); // new directory for asset
     var node = new Funnel(compassTree, {
-      srcDir: outputPath,
```

Comment:
Oh my goodness, I'm an idiot. You're actually right. It was a problem with my local copy. I'll add `srcDir` back in.